pypeman/remoteadmin.py

```diff
@@ -417,7 +416,7 @@ def do_channels(self, arg):
         result = self.client.channels()
         print("\nChannel list:")
         for idx, channel in enumerate(result):
-            print("{idx}) {name} ({status})".format(idx=idx, **channel))
+            print("{idx}) {short_name} ({status})".format(idx=idx, **channel))
```

Comment:
don't have to change.
but why not use f-strings?

Reply:
right, changed